docs/docs/migration-guide.mdx

```diff
@@ -57,35 +38,31 @@ rasa data convert core -f yaml --data={SOURCE_DIR} --out={TARGET_DIR}
 Converted files will have the same names as the original ones but with a
 `_converted.yml` suffix.
 
-Please note, that it's not currently possible to convert stories that contain forms
-automatically, these stories will be skipped and should be converted manually, see
-`Rule Policy` below.
-
 ### Policies
 
-With the introduction of [Rules](./rules.mdx), the following policies are
-deprecated:
+With the introduction of [rules](./rules.mdx) and the [RulePolicy](./policies.mdx#rule-policy),
+the following policies are deprecated:
 
 - [Mapping Policy](./policies.mdx#mapping-policy)
 - [Fallback Policy](./policies.mdx#fallback-policy)
 - [Two-Stage-Fallback Policy](./policies.mdx#two-stage-fallback-policy)
 - [Form Policy](./policies.mdx#form-policy)
 
-:::tip Automatic migration
-Rasa Open Source can take care of automatically migrating the Mapping Policy,
-the Fallback Policy and the Two-Stage-Fallback Policy for you. To do so, run the
-following command:
+
+To migrate the policies automatically, you can run the following command:
 
 ```bash
 rasa data convert config
 ```
 
-This command will take care of updating your `config.yml`, `domain.yml` and `rules.yml`, while
-making backups of your pre-existing files using the `.bak` suffix.
+This command will take care of updating your `config.yml`, ``domain.yml` and `rules.yml` while
+making backups of your existing files using the `.bak` suffix.
```

Comment:
```suggestion
This command will take care of updating your `config.yml` and``domain.yml`,
making backups of your existing files using the `.bak` suffix. It will also add a `rules.yml` if necessary.
```